packages/styles/scss/components/participant/_participant-tile.scss

```diff
@@ -6,17 +6,26 @@
   gap: 0.375rem;
   overflow: hidden;
   border-radius: var(--border-radius);
-  transition-property: box-shadow;
-  transition-timing-function: ease-in-out;
-  // When participant stops talking the transition longer and starts with a delay.
-  transition-delay: 0.5s;
-  transition-duration: 0.8s;
-  box-shadow: 0px 0px 0px 0 transparent;
-  &[data-speaking='true']:not([data-source='screen_share']) {
-    // When participant starts talking the transition short and without delay.
+  &::after {
```

Comment:
out of curiosity: why in `:after` vs. just on the class itself?

Reply:
`::after` overlays the tile and therefore does not change the size of the tile content. If we put it on the class directly it would scale the video when the indicator shows up.
Another way would be to go with a invisible border and then change the color or opacity on speaking. This can introduce alignment issues. For example, when you want the tile be flush against some "design"-line but the invisible border prevents that.

If we ever want to support switching between the outside and inside indicators. This approach would give us the flexibility to simply add twice the border width to the width and height of the :`:after` overlay to achieve an outside speaking indicator. To switch between the two, we could use a data-attr or property.